plugin_ide.ui/src-lang/melnorme/lang/ide/ui/editor/structure/EditorStructureUtil.java

```diff
@@ -11,28 +11,29 @@
 package melnorme.lang.ide.ui.editor.structure;
 
 import static melnorme.lang.ide.ui.EditorSettings_Actual.EDITOR_ID;
-import melnorme.lang.ide.ui.editor.EditorUtils;

```

Comment:
Yeah this is fine. We don't really look at imports, it's okay for Organize Imports action to reorder them. Most newer files are already organized that way, but that are some older files that haven't had the import headers organized yet.